packages/cfd/src/Containers/jurisdiction-modal/__test__/jurisdiction-card-back.spec.tsx

```diff
@@ -0,0 +1,76 @@
+        ).not.toBeInTheDocument();
+        expect(
+            screen.queryByText('A copy of your identity document (identity card, passport)')
+        ).not.toBeInTheDocument();
+        expect(screen.getByText('Your document is pending for verification.')).toBeInTheDocument();
+        expect(screen.getByText('Verification failed. Resubmit during account creation.')).toBeInTheDocument();
+        expect(screen.getByText('Your document is verified.')).toBeInTheDocument();
+    });
+
+    it('should render JurisdictionCardBack display required document_number and name_and_address submission', () => {
+        mock_props.verification_docs = ['document_number', 'name_and_address'];
+        render(<JurisdictionCardBack {...mock_props} />);
+        expect(screen.queryByText('A selfie of yourself.')).not.toBeInTheDocument();
+        expect(
+            screen.queryByText('A copy of your identity document (identity card, passport)')
+        ).not.toBeInTheDocument();
+        expect(screen.getByText('We need you to submit these in order to get this account:')).toBeInTheDocument();
+        expect(screen.getByText('Document number (identity card, passport)')).toBeInTheDocument();
+        expect(
+            screen.getByText(
+                'A recent utility bill (electricity, water or gas) or recent bank statement or government-issued letter with your name and address.'
+            )
+        ).toBeInTheDocument();
+        expect(screen.getByText('Your document is pending for verification.')).toBeInTheDocument();
+        expect(screen.getByText('Verification failed. Resubmit during account creation.')).toBeInTheDocument();
+        expect(screen.getByText('Your document is verified.')).toBeInTheDocument();
+    });
+
+    it('should render JurisdictionCardBack display required selfie, identity_document and name_and_address submission', () => {
+        mock_props.verification_docs = ['selfie', 'identity_document', 'name_and_address'];
+        render(<JurisdictionCardBack {...mock_props} />);
+        expect(screen.getByText('We need you to submit these in order to get this account:')).toBeInTheDocument();
+        expect(screen.getByText('A selfie of yourself.')).toBeInTheDocument();
+        expect(screen.getByText('A copy of your identity document (identity card, passport)')).toBeInTheDocument();
+        expect(
+            screen.getByText(
+                'A recent utility bill (electricity, water or gas) or recent bank statement or government-issued letter with your name and address.'
+            )
+        ).toBeInTheDocument();
+        expect(screen.getByText('Your document is pending for verification.')).toBeInTheDocument();
+        expect(screen.getByText('Verification failed. Resubmit during account creation.')).toBeInTheDocument();
+        expect(screen.getByText('Your document is verified.')).toBeInTheDocument();
+        expect(screen.queryByText('Document number (identity card, passport)')).not.toBeInTheDocument();
```

Comment:
here we are passing ['document_number', 'name_and_address'] as **verification_docs**
so shouldnt this statement `Document number (identity card, passport) `present in the document?